server/lib/field_hub/file_store.ex

```diff
@@ -4,6 +4,7 @@ defmodule FieldHub.FileStore do
   @valid_file_variants Application.compile_env(:field_hub, :valid_file_variants)
   @index_cache_name Application.compile_env(:field_hub, :file_index_cache_name)
   @index_cache_expiration_ms 1000 * 60 * 60 * 24
+  # @index_cache_expiration_ms 3000
```

Comment:
What did you try here? This line should probably be deleted at the end.

Reply:
Yes, I wanted to see what happened without wait one day.